SmartSyncExplorerReactNative/package.json

```diff
@@ -7,8 +7,8 @@
         "test": "jest"
     },
     "sdkDependencies": {
-        "salesforcemobilesdk-ios": "https://github.com/forcedotcom/SalesforceMobileSDK-iOS.git#dev",
-        "salesforcemobilesdk-android": "https://github.com/forcedotcom/SalesforceMobileSDK-Android.git#dev"
+        "SalesforceMobileSDK-iOS": "https://github.com/forcedotcom/SalesforceMobileSDK-iOS.git#dev",
```

Comment:
This was the only template using lowercase and it was causing path issues while copying resources. I lined it up to be consistent with the other templates.